org.eclipse.xtext/src/org/eclipse/xtext/resource/persistence/ResourceStorageLoadable.java

```diff
@@ -83,9 +83,8 @@ public InternalEObject loadEObject() throws IOException {
 		}.loadResource(resource);
 	}
 
```

Comment:
i assume we do this api breakage intentionally

Reply:
Yes, it was intentionally. If you think it's not worth it, I'll abandon the change.

Reply:
is not api anyway